### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by drpatelh**
See https://github.com/nf-core/atacseq/issues/218

### CODE REVIEW

**Comment by drpatelh** [comment]
Small changes that shouldn't be breaking the tests. They are failing due to download issues. I say we approve and merge this because tests will be re-run again anyway on the PR from `dev` to `master`

File: nextflow.config [hunk 1]
```diff
@@ -134,6 +131,7 @@ profiles {
     debug { process.beforeScript = 'echo $HOSTNAME' }
     conda {
         params.enable_conda    = true
+        conda.enabled          = true
```
Comment (JoseEspinosa) [inline nextflow.config:134]: The current version of the modules installed on the pipeline still uses the old syntax `conda (params.enable_conda ? "bioconda::tool=version" : null)`, won't then this change break the possibility of running the pipeline conda?

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Thanks @drpatelh!
Got a question about one of the changes implemented

**Comment by JoseEspinosa** [comment]
> Small changes that shouldn't be breaking the tests. They are failing due to download issues. I say we approve and merge this because tests will be re-run again anyway on the PR from `dev` to `master`

These download errors are more are more usual, by any chance do you know what's the reason behind them?

File: nextflow.config (hunk 1 above)
Comment (drpatelh) [reply nextflow.config:134]: Nope. It shouldn't do. It will just mean that it works with later versions of Nextflow.

**Comment by drpatelh** [comment]
> These download errors are more and more usual, by any chance do you know what's the reason behind them?

We could move the data to S3 instead of being hosted on Github. I did this for nf-core/rnaseq and the tests were more stable. We could do the same here and upload the test data to the nf-core AWS account. I can take a look at this in the week.

For now this PR should be good to go so we can test `-profile test_full` again to make sure it is working on AWS.

**Review by maxulysse — APPROVED** [review]

File: nextflow.config (hunk 1 above)
Comment (JoseEspinosa) [reply nextflow.config:134]: I dont know why I have read the change as you also deleted `params.enable_conda = true` :sweat:

**Pull request merged**